src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -159,6 +160,7 @@ class ReferralService(
 
     val sentReferral = referralRepository.save(referral)
     eventPublisher.referralSentEvent(sentReferral)
+    supplierAssessmentService.createInitialAssessment(referral, user)
```

Comment:
naming issue here - 'initial assessment' isn't in use, i think it's just 'supplier assessment' ?